core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -133,6 +133,16 @@ private SqlLibraryOperators() {
   public static final SqlFunction DATEADD =
       new SqlTimestampAddFunction("DATEADD");
 
+  /** The "DATE_ADD(start_date, num_days)" function
+   * (Spark) Returns the date that is num_days after start_date. */
+  @LibraryOperator(libraries = {SPARK})
+  public static final SqlFunction DATE_ADD_SPARK =
```

Comment:
DATE_ADD is enough. The LiberaryOperator will let us know if it follows the Spark style.

Reply:
@NobiGo Bigquery's DATE_ADD already exists. Duplicate naming will cause conflicts.